slides/2-html-css.html

```diff
@@ -1240,6 +1318,22 @@ <h2><i class="fas fa-paint-brush"></i> Flexbox</h2>
 
                     </textarea>
                 </section>
+                <section data-markdown style="font-size: 0.75em;">
+                    <textarea data-template>
+                        ## <i class="fas fa-hand-paper"></i> Grid
+
+                        In some scenarios, the grid layout is more convenient,
```

Comment:
Added content : Mention of grid, I believe in some contexts it is preferable than the old and sometimes messy flexbox. In particular, I will show the `grid-template-area` property and how cool it is for responsive designs.